mysql-test/suite/galera/t/galera_flush_table_for_export.test

```diff
@@ -0,0 +1,41 @@
+# ==== Purpose ===
+#
+# This test verifies that server performs proper cleanup when the session was
+# disconnected/killed while there is an active FLUSH TABLE FOR EXPORT in
+# effect.
+#
+# === References ===
+#
+# PXC-3739: FLUSH TABLES FOR EXPORT lock not released when session ends
+
+--source include/galera_cluster.inc
+--source include/count_sessions.inc
+
+# Create auxiliary connections for the test, which will be disconnected later.
+--connect(con1,localhost,root,,test, $NODE_MYPORT_1)
+--connect(con2,localhost,root,,test, $NODE_MYPORT_1)
+CREATE TABLE t1 (i INT PRIMARY KEY);
+
+# Test-1: Execute FLUSH TABLE for EXPORT and disconnect the connection.
+FLUSH TABLE t1 FOR EXPORT;
+--disconnect con2
+
+# Test that INSERT works.
+--connection node_1
+INSERT INTO t1 VALUES(1);
+
+# Test-2: Execute FLUSH TABLE for EXPORT and kill the session from another session.
+--connection con1
+--let $conn_id = `SELECT CONNECTION_ID();`
```

Comment:
please check if normal FLUSH TABLE t1 is also affected?

Reply:
Looks like the commands FLUSH TABLE t1 and LOCK TABLE t1 have no effect on the provider.